Makefile

```diff
@@ -144,7 +144,9 @@ dist: manpage $(SPEC_FILE) $(CLIB_HEADER)
 		cd rust; \
 		cargo vendor-filterer \
 			--platform x86_64-unknown-linux-gnu \
```

Comment:
I'd recommend the declarative spec in `Cargo.toml` BTW and also update to use `tier = "2"`, xref e.g. https://github.com/containers/bootc/blob/41ebc7f019e425b7bf43f2e6307788b34c4e2046/Cargo.toml#L12

Reply:
Thanks for the suggestion! Will try